index.html

```diff
@@ -12,15 +12,20 @@
 </head>
 <body>
     <header>
-            <div>
+
+        <nav class="hamburger">
                 <a class="logo" href="#">Welcome</a>
-            </div>
-            <nav class="hamburger">
-                <img src="media\Union.svg" alt="hamburger">
+                <ul class="link-desktop">
+                    <li class="links-style"><a href="#">Portoflio</a></li>
+                    <li class="links-style"><a href="#">About</a></li>
+                    <li class="links-style"><a href="#">Contact</a></li>
+                </ul>
+                <img class="hamburger-img" src="media\Union.svg" alt="hamburger">
             </nav>
     </header>
 
     <section class="intro">
+            <div class="intro-desktop">
             <h1>Hey there! I'm Ferizad</h1>
             <h2>I am a software developer</h2>
             <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
```

Comment:
- [ ] great job. please kindly add the social icons on the headline section of the desktop version so it follows the figma design
yours
![Screenshot (823)_LI](https://user-images.githubusercontent.com/65298048/220934559-e9ebad95-9f5e-45e6-9016-7bdefda1987a.jpg)
figma
![Screenshot (824)_LI](https://user-images.githubusercontent.com/65298048/220934706-5dec4d1a-e17e-4359-882d-6e921cdb5bcb.jpg)